payments-app-extension-credit-card/shopify.extension.toml.liquid

```diff
@@ -12,7 +12,11 @@ payment_session_url = "https://api.paymentprovider.com/endpoint"
 refund_session_url = "https://api.paymentprovider.com/refund"
 capture_session_url = "https://api.paymentprovider.com/capture"
 void_session_url = "https://api.paymentprovider.com/void"
+# All countries where your app is available for merchants to install. Learn more:
+# https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
```

Comment:
Let's use the official iso standard website rather than Wikipedia.
```suggestion
# List of ISO 3166 (alpha-2) country codes your app is available for installation by merchants. Learn more:
# https://www.iso.org/iso-3166-country-codes.html
```